mailpoet/assets/js/src/automation/editor/components/actions/trash-button.tsx

```diff
@@ -30,7 +31,12 @@ export function TrashButton(): JSX.Element {
         onCancel={() => setShowConfirmDialog(false)}
         __experimentalHideHeader={false}
       >
-        You are about to delete the “{workflow.name}” workflow.
+        {sprintf(
+          __('You are about to delete the “%s” workflow.', 'mailpoet'),
```

Comment:
In https://a8c.slack.com/archives/C0313447YBC/p1666334421957099 we used the simple non-typographic quotes (`"`) for now. Not sure if that's a good decision, we don't have a super-strong argument either way but we did what WC does at the moment.

Reply:
Addressed in 9c8236cf1